src/components/style-selector/medium-style-selector.vue

```diff
@@ -0,0 +1,50 @@
+<script setup lang="ts">
+import { useTranslation } from 'i18next-vue'
+import { computed } from 'vue'
+import { storeToRefs } from 'pinia'
+
+import { getDefaultMediumStyling } from '@/services/styles/styles.service'
+import MediumStyleItem from './medium-style-item.vue'
+import { StyleItem } from '@/composables/mvt-styles/mvt-styles.model'
+import { Layer } from '@/stores/map.store.model'
+import { useStyleStore } from '@/stores/style.store'
+
+const styleStore = useStyleStore()
+const { bgStyle } = storeToRefs(styleStore)
+
+const { t } = useTranslation()
+const props = defineProps<{
+  layer: Layer
+}>()
+const isColorVisible = computed(
+  () => props.layer.name === 'basemap_2015_global'
```

Comment:
Maybe use a `const` here ? (We already have `DEFAULT_BGLAYER_NAME` in the persistor service)

Reply:
I would prefer not to use that variable.
Whether the colors can be edited does not depend on the default layer. We could use the simple_style_class of the background layers fixture which I introduse for the external editor (https://github.com/Geoportail-Luxembourg/luxembourg-geoportail/blob/GSLUX-593_external_editor/src/__fixtures__/background.config.fixture.ts#L7), or even add a property "edit_colors"??

Anyways, the logic for default background layers is quite dispersed between different modules. This was the case in v3.
I would suggest grouping default layer logics somewhere. Maybe a new dedicated service, what do you think, @AlitaBernachot ?

Reply:
Ok, or maybe just define `basemap_2015_global` as a constant within the component, would already allow extracting the hard coded value from the `computed` and improve readability.